.gitignore

```diff
@@ -231,3 +231,6 @@ detox/test/artifacts
 Detox-android/
 Detox-ios-src.tbz
 Detox-ios.tbz
+
+# Git VCS 
+website/**/*.css
```

Comment:
VCS?

Also, why do we need that under .gitignore?

Reply:
@moutro , I'd move it before `## Final outputs` closer to `/examples/...`.
Also, you can proceed `/website` with a slash in the beginning.

@asafkorem, this is correct. We migrated to SCSS. All the resulting CSS files are just temporary artifacts needed for serving and deploying the website.

Reply:
Done. Thanks!